analyzers/SpamhausDBL/SpamhausDBL.json

```diff
@@ -0,0 +1,15 @@
+{
+    "name": "SpamhausDBL",
+    "version": "1.0",
+    "author": "Wes Lambert",
+    "url": "https://github.com/TheHive-Project/Cortex-Analyzers",
+    "license": "AGPL-V3",
+    "description": "Perform domain lookup to Spamhaus DBL",
+    "dataTypeList": ["domain"],
```

Comment:
Is it meaningful to add `fqdn` to that list? I don't know SpamhausDBL.

Reply:
Certainly, I'll look into that. 👍

Reply:
`fqdn` has been added ➕.